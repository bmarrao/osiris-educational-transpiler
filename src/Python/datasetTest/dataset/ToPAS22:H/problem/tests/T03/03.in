5
4 5
AAAAAAAAAAAAAAAAAAAA BBBBBBBBBBBBBBBBBBBB CCCCCCCCCCCCCCCCCCCC DDDDDDDDDDDDDDDDDDDD
CCCCCCCCCCCCCCCCCCCC BBBBBBBBBBBBBBBBBBBB
AAAAAAAAAAAAAAAAAAAA DDDDDDDDDDDDDDDDDDDD
CCCCCCCCCCCCCCCCCCCC DDDDDDDDDDDDDDDDDDDD
AAAAAAAAAAAAAAAAAAAA BBBBBBBBBBBBBBBBBBBB
DDDDDDDDDDDDDDDDDDDD BBBBBBBBBBBBBBBBBBBB
4 4
ke yMk xo OnCQZ
yMk ke
xo ke
yMk xo
xo OnCQZ
4 5
eAUDQ SzTe vMi cZAS
eAUDQ vMi
SzTe eAUDQ
eAUDQ cZAS
SzTe vMi
SzTe cZAS
4 0
okct JusG lj pU
4 1
YnNJ Uqa g j
g j
