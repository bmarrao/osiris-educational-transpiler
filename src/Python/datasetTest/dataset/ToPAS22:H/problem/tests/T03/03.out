Impossivel
Impossivel
Impossivel
Possivel: 24
Possivel: 16
